internal/config/config.go

```diff
@@ -35,6 +36,39 @@ type versionSetting struct {
 
 type Engine string
 
+type Paths []string
+
+func (p *Paths) UnmarshalJSON(data []byte) error {
```

Comment:
I am not sure that I need this function. When the tests were failing they contained yaml parsing error even when the tests appeared to use json. Still trying to figure that out, but until I understand that there is some conversion or something going on then I will leave this here for now.

Reply:
We technically don't need this, as we use the yaml package to parse both JSON and YAML. However, I like keeping this so that we ensure that the configuration can still be parsed with the `encoding/json` package.